app/src/main/res/xml/pref_advanced_settings.xml

```diff
@@ -375,6 +375,12 @@
                 android:key="xdrip_webservice_open"
                 android:summary="Accept connections from any network instead of just internally within this device. There are security implications to enabling this!"
                 android:title="Open Web Service" />
+            <EditTextPreference
+                android:defaultValue="ABC123"
```

Comment:
should default to `""` to unless we make it only require the secret when the server is set to open but I'm thinking its better if it defaults to no secret and then has consistent behavior whether localhost or not and also can allow restriction of the localhost interface.